InferenceSystem/infra/CreateSPforResourceGroup.ps1

```diff
@@ -0,0 +1,58 @@
+./Write-Log.ps1
+<#
+[AI4Orca l Hack Project]
+AI4Orca Hack Project resource provisioning
+1. Create Service Principal for GitHub Actions
+  
+#>
+
+
+# Pre-request:  Azure ML CLI : az extension add --name azure-cli-ml
+
+Write-Host  "==== AI4Orca Hack Project -  Create Service Principal Services  ===="
+Write-Host   "======================================================" 
+
+#Get Global Config
+$config = Get-Content .\provision-config.json | ConvertFrom-Json
+
+#Start to deploy Azure Resources
+Write-Host  "Before deployment, please make sure you have installed the Powershell Core (version 6.x up) and latest Azure Cli"
+
+az login
+az account set --subscription $config.AzureSubscriptionId
+
+$rg=az group show -n $config.ResourceGroupName | ConvertFrom-Json
+
+# Check Resource Group exists
+if($rg -eq $null)
+{
+    Write-Output ("Resource Group $($config.ResourceGroupName) not exists")
+    Write-Output ("Creating/Assign Resource Group for Deployment")
+    az group create -n $config.ResourceGroupName -l $config.Location
+}
+else
+{
+    Write-Output ("Resource Group $($rg.name) exists")
+}
+
+# Create the service principal with rights scoped to the subscription.
+#$SP_PASSWD=az ad sp create-for-rbac --name $config.SERVICE_PRINCIPAL_NAME --scopes --scopes /subscriptions/$($config.AzureSubscriptionId)/resourceGroups/$($config.ResourceGroupName) --role contributor --sdk-auth --query password --output tsv
+#$SP_PASSWD=az ad sp create-for-rbac --name $config.SERVICE_PRINCIPAL_NAME --scopes --scopes /subscriptions/$($config.AzureSubscriptionId)/resourceGroups/$($config.ResourceGroupName) --role contributor --sdk-auth --query password --output tsv
+#$SP_PASSWD=az ad sp create-for-rbac --name $config.SERVICE_PRINCIPAL_NAME --scopes $ACR_REGISTRY_ID --role contributor --query password --output tsv
```

Comment:
Is it some unneeded leftover code from iteration? At least the first two commented lines are identical